attest/verifier/build.rs

```diff
@@ -289,3 +306,105 @@ fn write_signing_key(signer_key_path: impl AsRef<Path>, signer_key: &SigningKey)
     )
     .expect("Could not write signer key PEM to file");
 }
+
+/// Returns the mandatory DCAP OID extensions, `(oid, bytes)`
+///
+/// The extensions (and OID values) are documented at
+/// <https://api.trustedservices.intel.com/documents/Intel_SGX_PCK_Certificate_CRL_Spec-1.5.pdf#%5B%7B%22num%22%3A193%2C%22gen%22%3A0%7D%2C%7B%22name%22%3A%22XYZ%22%7D%2C69%2C690%2C0%5D>
+fn dcap_extensions() -> (Vec<u8>, Vec<u8>) {
```

Comment:
I'm up for suggestions on how this might be better broken up it's a bit long winded of a function.